extension/src/popup/components/warningMessages/FlaggedWarningMessage.tsx

```diff
@@ -56,9 +56,9 @@ const MemoWarningMessage = ({ isMemoRequired }: { isMemoRequired: boolean }) =>
       subheader="Memo is required"
     >
       <p>
-        The destination account (Kraken) requires the use of the memo field
-        which is not present in the transaction you're about to sign. Freighter
-        automatically disabled the option to sign this transaction.
+        A destination account requires the use of the memo field which is not
+        present in the transaction you’re about to sign. Freighter automatically
+        disabled the option to sign this transaction.
```

Comment:
Updating copy to reflect that one (of possibly several operations) threw an error